api/inc/api.h

```diff
@@ -85,6 +85,8 @@ typedef struct {
     void (*spin_lock)(UvisorSpinlock * spinlock);
     void (*spin_unlock)(UvisorSpinlock * spinlock);
 
+    void (*debug_semihosting_enable)(void);
```

Comment:
please add comment stating that it is ONLY intended to be called by debugger after reset